src/ethereum_test_specs/blockchain.py

```diff
@@ -316,7 +316,6 @@ class BlockchainTest(BaseTest):
     blocks: List[Block]
     genesis_environment: Environment = Field(default_factory=Environment)
     verify_sync: bool = False
-    chain_id: int = 1
```

Comment:
Wait we already had this in the blockchain test?
I think it's safer to reuse this, because it's not really part of the environment, but more of the chain configuration.
Same for the state test, it should be a property of `state_test`, not environment.

Reply:
Also, the value has to be copied over on https://github.com/ethereum/execution-spec-tests/blob/061e41cce851e5d4d7de1d0612c84349f78b492e/src/ethereum_test_specs/state.py#L95

Reply:
Ah yes this makes more sense! For some reason I thought it would be easier to parameterize the chain id this way but definitely not.

1) What about moving it up to BaseTest (on the assumption that all tests required a chain id, which I feel is safe)?

2) This has me thinking about moving `pre, post, env` from State/BlockchainTest into BaseTest as well. This would work for now but would force future tests to include these fields. A future TransactionTest might not want these fields.

Reply:
Yes I think it might be a good idea, there might be some issues I'm not seeing right now but it should be straight-forwards.

Reply:
I completed 1) here (moving it up to base test). Skipping 2) as transaction tests will make it redundant.